deeplabcut/utils/plotting.py

```diff
@@ -177,61 +177,82 @@ def plot_trajectories(
     linewidth=1.0,
     track_method="",
 ):
-    """
-    Plots the trajectories of various bodyparts across the video.
+    """Plots the trajectories of various bodyparts across the video.
 
     Parameters
     ----------
-     config : string
-    Full path of the config.yaml file as a string.
+    config: str
+        Full path of the config.yaml file.
 
-    videos : list
-        A list of strings containing the full paths to videos for analysis or a path to the directory, where all the videos with same extension are stored.
+    videos: list[str]
+        Full paths to videos for analysis or a path to the directory, where all the
+        videos with same extension are stored.
 
-    videotype: string, optional
-        Checks for the extension of the video in case the input to the video is a directory.\n Only videos with this extension are analyzed.
-        If left unspecified, videos with common extensions ('avi', 'mp4', 'mov', 'mpeg', 'mkv') are kept.
+    videotype: str, optional, default=""
+        Checks for the extension of the video in case the input to the video is a
+        directory. Only videos with this extension are analyzed.
+        If left unspecified, videos with common extensions
+        ('avi', 'mp4', 'mov', 'mpeg', 'mkv') are kept.
 
-    shuffle: list, optional
-    List of integers specifying the shuffle indices of the training dataset. The default is [1]
+    shuffle: list, optional, default=1
+        List of integers specifying the shuffle indices of the training dataset.
```

Comment:
Just noticed that the doc was originally wrong here. It should read:
```suggestion
    shuffle: int, optional, default=1
        Integer specifying the shuffle index of the training dataset.
```